Web/Cookie.hs

```diff
@@ -246,7 +245,7 @@ formatCookieExpires =
 
 parseCookieExpires :: S.ByteString -> Maybe UTCTime
 parseCookieExpires =
-    fmap fuzzYear . parseTime defaultTimeLocale expiresFormat . S8.unpack
+    fmap fuzzYear . parseTimeM True defaultTimeLocale expiresFormat . S8.unpack
```

Comment:
`parseTime` is deprecated. Its implementation is:

`parseTime = parseTimeM True`

https://github.com/haskell/time/blob/37dad65c7f45df70a9f377428626d2368f0458ec/lib/Data/Time/Format/Parse.hs#L164

Reply:
Note: this change requires time-1.5. It's been out for three years, so hopefully that's ok?

Reply:
Yeah, I'm cool with that. Thanks for doing this!